steps/create-a-new-release-in-jira/1.0.0/step.yml

```diff
@@ -0,0 +1,100 @@
+title: Create a new release in JIRA
+summary: |
+  Create a new release in JIRA.
+description: |
+  This steps create a Release in your JIRA project, with the start date, publication and description you need.
+website: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira
+source_code_url: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira
+support_url: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira/issues
+published_at: 2021-10-06T21:03:16.937402-03:00
+source:
+  git: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira.git
+  commit: 567ef82a59926847431c10b3256274ac2ad35bb9
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      Jira project prefix to search for tasks.
+
+      e.g. "PROJ"
+    is_expand: true
+    is_required: true
+    summary: Jira project prefix to search for tasks.
+    title: Jira project prefix
+  project_prefix: null
+- jira_domain: null
+  opts:
+    description: |
+      Jira company domain.
+
+      e.g. "example.atlassian.net"
+    is_expand: true
+    is_required: true
+    summary: Jira company domain
+    title: Jira domain
+- jira_user: null
+  opts:
+    description: |
+      Jira user used to authenticate with Jira API
+
+      e.g. "someone@example.pl"
+    is_sensitive: true
+    summary: Jira user used to authenticate with Jira API
+    title: Jira user
+- jira_token: null
+  opts:
+    description: |
+      Jira user token used to authenticate with Jira API
+
+      24 character string
+    is_sensitive: true
+    summary: Jira user token used to authenticate with Jira API
+    title: Jira user token
+- opts:
+    description: |
+      Version description.
+
+      e.g. "An excellent version"
+    is_expand: true
+    is_required: true
+    summary: Version description
+    title: Version  description
+  version_description: null
+- opts:
+    description: |
+      Version name.
+
+      e.g. "v1.0.0"
+    is_expand: true
+    is_required: true
+    summary: Version name
+    title: Version  name
+  version_name: null
+- opts:
+    description: |
+      If the version was published.
+    is_required: true
+    summary: If the version was published
+    title: If the version was published?
```

Comment:
Description of this input does not seem to match the logic behind it.
Past tense here suggests that was already released. Such release event seems to be unrelated to the action performed by the step.
However, in the script you are assigning today's date. So it suggests that the release happens at the time of step execution, not in the past.

You should correct either of that places.